components/gallery/useGalleryItem.ts

```diff
@@ -96,9 +96,15 @@ export const useGalleryItem = (nftId?: string): GalleryItem => {
     nft.value = nftEntity
 
     const resources = nftEntity.resources?.map((resource) => {
+      const imageSrc =
+        resource.meta?.animationUrl ||
+        resource.src ||
+        resource.meta?.image ||
+        resource.thumb
+
```

Comment:
not part of this PR for sure
but why do we have so many possible sources?

@vikiival

Reply:
Because Bruno said so 

https://github.com/rmrk-team/rmrk-spec/blob/master/standards/rmrk2.0.0/entities/nft.md#resources-and-resource